pkg/overlay/config.go

```diff
@@ -34,16 +34,19 @@ type NodeSelectionConfig struct {
 	OnlineWindow      time.Duration `help:"the amount of time without seeing a node before its considered offline" default:"1h"`
 	DistinctIP        bool          `help:"require distinct IPs when choosing nodes for upload" releaseDefault:"true" devDefault:"false"`
 
-	AuditReputationRepairWeight  float64 `help:"weight to apply to audit reputation for total repair reputation calculation" default:"1.0"`
-	AuditReputationUplinkWeight  float64 `help:"weight to apply to audit reputation for total uplink reputation calculation" default:"1.0"`
-	AuditReputationAlpha0        float64 `help:"the initial shape 'alpha' used to calculate audit SNs reputation" default:"1.0"`
-	AuditReputationBeta0         float64 `help:"the initial shape 'beta' value used to calculate audit SNs reputation" default:"0.0"`
-	AuditReputationLambda        float64 `help:"the forgetting factor used to calculate the audit SNs reputation" default:"1.0"`
-	AuditReputationWeight        float64 `help:"the normalization weight used to calculate the audit SNs reputation" default:"1.0"`
+	AuditReputationRepairWeight float64 `help:"weight to apply to audit reputation for total repair reputation calculation" default:"1.0"`
+	AuditReputationUplinkWeight float64 `help:"weight to apply to audit reputation for total uplink reputation calculation" default:"1.0"`
+	AuditReputationAlpha0       float64 `help:"the initial shape 'alpha' used to calculate audit SNs reputation" default:"1.0"`
+	AuditReputationBeta0        float64 `help:"the initial shape 'beta' value used to calculate audit SNs reputation" default:"0.0"`
+	AuditReputationLambda       float64 `help:"the forgetting factor used to calculate the audit SNs reputation" default:"1.0"`
+	AuditReputationWeight       float64 `help:"the normalization weight used to calculate the audit SNs reputation" default:"1.0"`
+	// TODO(nat): figure out default for DQs
```

Comment:
Are we going to know this before merging?

Reply:
Yes, we are waiting on some initial values from Dr. Ben